lightning/src/ln/onion_route_tests.rs

```diff
@@ -270,9 +278,10 @@ fn test_fee_failures() {
 	claim_payment(&nodes[0], &[&nodes[1], &nodes[2]], payment_preimage_success);
 
 	let (_, payment_hash, payment_secret) = get_payment_preimage_hash!(nodes[2]);
+	let short_channel_id = channels[0].0.contents.short_channel_id;
 	run_onion_failure_test("fee_insufficient", 0, &nodes, &route, &payment_hash, &payment_secret, |msg| {
 		msg.amount_msat -= 1;
-	}, || {}, true, Some(UPDATE|12), Some(NetworkUpdate::ChannelClosed { short_channel_id: channels[0].0.contents.short_channel_id, is_permanent: true}));
+	}, || {}, true, Some(UPDATE|12), Some(NetworkUpdate::ChannelClosed { short_channel_id, is_permanent: true}), None);
```

Comment:
I'd expect `fee_insufficient` to include an SCID, especially if we're marking the channel as closed - a few things here:
 * are we generating invalid fee_insufficient responses (ie not including the ChannelUpdate object)?
 * whether we are or not, we probably should have a SCID to blame if we get a failure with UPDATE but no ChannelUpdate.